setup.py

```diff
@@ -1,120 +0,0 @@
-import os
```

Comment:
don't we need this anymore? at all?

Reply:
some pretty old RHEL might need a setup.py entrypoint for re-packaging. I added the minimalist setup.py needed. With this, all cases are covered.

Reply:
If you set the minimum setuptools version to 64.0.0, no `setup.py` file is required.

Isn't it an issue on their end if they package the library with an incompatible version of setuptools? I guess it's good to keep the `setup.py` stub for extra safety

Reply:
Done as well.